app/components/open_project/common/submenu_component.sass

```diff
@@ -52,6 +56,7 @@
     font-size: 12px
     cursor: pointer
     width: 100%
+    font-weight: var(--base-text-weight-semibold, 600)
```

Comment:
I am not a big fan of this change, I admit.. I see the need to distinguish the headline from the other elements now that they have the same color, but the bold font looks weird imho. Please double check with @psatyal for this.